azkaban-common/src/test/java/azkaban/server/session/SessionCacheTest.java

```diff
@@ -32,6 +32,10 @@ SessionCache freshSessionCache() throws Exception {
     return new SessionCache(props);
   }
 
+  SessionCache makeSessionCacheWithLongTime() throws Exception {
```

Comment:
how about making session.time.to.live as an argument? E.g:
`createSessionCacheWithTTL(long ttl)`,
 sessionCacheFindByIP will pass a longer ttl
for rest test cases, passing a shorter ttl like 200 ms?

It's less confusing if ttl is explicitly set.

Reply:
I do refactoring makeSessionCacheWithLongTimeto createSessionCacheWithTTl. Please check it.

Reply:
Thanks, sorry for back & forth,  should SessionCacheHit, SessionCacheCount also use long TTL to avoid being flaky?

Reply:
I think it's more safe than before that use shortTTL. Would I do refactoring from this code to use longTTL above two methods?

Reply:
I changed from shortTTL to longTTL in SessionCacheHit and SessionCacheCount methods.
Please check it. Thanks.

Reply:
thanks!